src/amchart/Candle.js

```diff
@@ -13,13 +13,12 @@
         
         this._gType = "candlestick";
         
-        this._openField;
-        this._closeField;
-        this._categoryField;
-        this._lowField;
-        this._highField;
-    };
-    
+        this._openField = undefined;
+        this._closeField = undefined;
+        this._categoryField = undefined;
+        this._lowField = undefined;
+        this._highField = undefined;
```

Comment:
If you want to init them - then open a diff PR - the goal of this PR is to be JSHint compliant with no logic changes...